README.md

```diff
@@ -1,8 +1,28 @@
 This is a [Next.js](https://nextjs.org/) project bootstrapped with [`create-next-app`](https://github.com/vercel/next.js/tree/canary/packages/create-next-app).
 
+## Tools
+- Package Manager / Bundling
+  - [Bun Package Manager](https://bun.sh/) 
+- Components / Styling
+  - [tailwindcss](https://tailwindcss.com/)
+  - [shadcn/ui](https://ui.shadcn.com/)
+- Databse
+  - [convex](https://www.convex.dev/)
+
 ## Getting Started
 
-First, install packages:
+First install Bun:
+
+```bash
+curl -fsSL https://bun.sh/install | bash
+```
```

Comment:
Nice! I would also link to the bun.sh docs here in case the install command were to change in the future ( I doubt it, but I think that would be good to link here anyways )